pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -48,9 +48,12 @@ buildPythonPackage rec {
         url = "https://github.com/openai/triton/commit/52c146f66b79b6079bcd28c55312fc6ea1852519.patch";
         hash = "sha256-098/TCQrzvrBAbQiaVGCMaF3o5Yc3yWDxzwSkzIuAtY=";
       })
+
+      # Upstream startded pinning CUDA version and falling back to downloading from Conda
+      # in https://github.com/triton-lang/triton/pull/1574/files#diff-eb8b42d9346d0a5d371facf21a8bfa2d16fb49e213ae7c21f03863accebe0fcfR120-R123
+      ./0000-dont-download-ptxas.patch
```

Comment:
To keep the track of the history, the pin was introduced in https://github.com/NixOS/nixpkgs/pull/258328/files#diff-7c765cc9768cdae049f8712b4ef719bc94ec8ec2e8df2dd50ddf14c6acd4e580R8354; presumably because of the upstream change, but idk

Reply:
It was already there, just guarded by `!cudaSupport`

Reply:
ohhh i see now. got it!